.gitignore

```diff
@@ -1,3 +1,4 @@
+/.idea
```

Comment:
remove this please, it is not project related

you can use https://help.github.com/articles/ignoring-files/#create-a-global-gitignore for that on your local machine :)